include/dlaf/util_blas.h

```diff
@@ -0,0 +1,146 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2020, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <blas.hh>
+
+#include <dlaf/types.h>
+
+namespace dlaf {
+namespace util {
```

Comment:
I would put these methods in dlaf::tile::internal as I think that they are not part of the public API.

Reply:
Yes, agreed! I'll change that.

Reply:
Done.